aws-s3-public-bucket/variables.tf

```
@@ -0,0 +1,46 @@
+variable "owner" {
+  type        = string
+  description = "Owner for tagging and naming."
+}
+
+variable "project" {
+  type        = string
+  description = "Project for tagging and naming."
+}
+
+variable "service" {
+  type        = string
+  description = "Service for tagging and naming."
+}
+
+variable "env" {
+  type        = string
+  description = "Env for tagging and naming."
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "The name of the bucket."
```

```suggestion
  description = "The name of the bucket. Note that `-public` will be appended to `bucket_name`s that don't contain a `public` substring. This module will output the computed `bucket_name`."
```
Or something like that so users know to potentially expect a change in bucket name. What do you think?